schemas/v1.0/azure.yaml.json

```
@@ -27,6 +27,31 @@
                 }
             }
         },
+        "infra": {
+            "type": "object",
+            "additionalProperties": true,
+            "properties": {
+                "provider":{
+                    "type": "string",
+                    "title": "Type of infrastructure provisioning provider",
+                    "description": "Optional. When specified will set the infrastructure provisioning provider used to provision the Azure resources for the application. (Default: bicep)",
```

"When specified will set" seems redundant. Maybe move it to the end, if we want to keep it? 

```suggestion
                    "description": "Optional. The infrastructure provisioning provider used to provision the Azure resources for the application. (Default: if unset bicep)",
```


>